docs/release-notes/mongoid-7.4.txt

```diff
@@ -39,6 +39,32 @@ Mongoid 7.4 deprecates support for MongoDB 3.4 and earlier.
 Mongoid 8 will require MongoDB 3.6 or newer.
 
 
+Feature Flags Summary
+---------------------
+
+To ensure a stable upgrade path from Mongoid 7.3, Mongoid 7.4
+introduces feature flags which are further explained in the
+sections below.
+
+To enable all new behavior in Mongoid 7.4, please use the following
+configuration. We recommend newly created apps to do this as well.
+
+.. code-block:: ruby
+
+  # in an initializer
+
+  # Enable all new behavior in Mongoid 7.4
+  Mongoid.legacy_triple_equals  = false
+  Mongoid.object_id_as_json_oid = false
+  Mongoid.compare_time_by_ms    = true
+  Mongoid.broken_aggregables    = false
+  Mongoid.broken_updates        = false
+  Mongoid.broken_and            = false
+  Mongoid.broken_scoping        = false
+  Mongoid.broken_alias_handling = false
+  Mongoid.legacy_pluck_distinct = false
```

Comment:
Hi @johnnyshields ,

I merged most of this PR as https://github.com/mongodb/mongoid/pull/5247 except for this section.

While it is possible to configure Mongoid programmatically in the way shown here, our documentation (https://www.mongodb.com/docs/mongoid/master/reference/configuration/) instructs users to configure Mongoid via the configuration file. If you change this section to use the same language as https://www.mongodb.com/docs/mongoid/master/reference/configuration/#mongoid-configuration-options that could be merged as well.

Reply:
Thanks! I raised https://github.com/mongodb/mongoid/pull/5250 for the follow-up